Dockerfile

```
@@ -29,7 +29,22 @@ WORKDIR /usr/local/src
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
+RUN apt update
+RUN apt -y install tesseract-ocr
```

For container image layer economy, a suggestion:
```suggestion
RUN apt update && \\
    apt -y install tesseract-ocr tesseract-ocr-nld tesseract-ocr-spa
```

---

I prefer adding languages to be on a separate layer to avoid having to rebuild from that step onwards.